server/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You don't even need a .gitignore here. Make one at parent level and ignore node modules, it will ignore for both the react app and the server. Also, can you test the gitignore? Because this is one repository, I"m not sure if they work at the sub directory level?
Irrespective make one common one at root. You don't need this file for one line